src/hyperfine/benchmark.rs

```diff
@@ -74,13 +78,15 @@ pub fn time_shell_command(
         time_real = subtract_shell_spawning_time(time_real, spawning_time.time_real);
         time_user = subtract_shell_spawning_time(time_user, spawning_time.time_user);
         time_system = subtract_shell_spawning_time(time_system, spawning_time.time_system);
+        max_rss = subtract_shell_spawning_time(max_rss, spawning_time.max_rss);
```

Comment:
Okay, so this subtracts the memory required by the shell? In this case, we should probably give the function a new name?

Reply:
Indeed, I did the least amount of changes to get something going, if you have idea on function names please tell me :)